### DISCUSSION TITLE
Biogenic Carbon Flux Module Simulation Results in SUEWS

### DISCUSSION THREAD

**Discussion by yyssqq** [UMEP-dev/UMEP#603] [Q&A]
Dear SUEWS Team,

I am currently using the SUEWS model, version supy 2024.3.25.dev0, to simulate the impact of urban vegetation on the carbon cycle. I noticed on your documentation page for the Biogenic Carbon Flux Module (https://suews.readthedocs.io/en/latest/input_files/SUEWS_SiteInfo/SUEWS_BiogenCO2.html) that it is not yet ready for use. However, upon reviewing the model's source code on GitHub, I found that the suews_phys_biogenco2.f95 module may be set up. Therefore, I conducted simulations using the Sample data(df_state_init, df_forcing = sp.load_SampleData()), only altering the EmissionsMethod parameter in Runcontrol.nml to activate the BiogenCO2 module. I have attached the results of these simulations.
[EmissionsMethod_Results.zip](https://github.com/UMEP-dev/UMEP/files/15069992/EmissionsMethod_Results.zip)

Here are my observations:

With EmissionsMethod set to 15, Fc, FcPhoto, FcRespi, FcMetab, FcTraff, and FcBuild are computed, but FcRespi remains constant at 0.4617. I am unsure why this is happening.
With EmissionsMethod set to 25, the results are the same as with EmissionsMethod 15, with no variation in values.
With EmissionsMethod set to 35, all the same components are calculated, but FcPhoto is consistently positive (which seems unusual), and FcRespi remains constant at 0.4617.
With EmissionsMethod set to 45, again all components are computed, but FcPhoto is 0, and FcRespi remains constant at 0.4617.
I have two main questions:

1）Whether the now-released modeling code correctly simulates biological carbon fluxes?
2）Are there any parameters I might have set incorrectly, or are there additional settings I should consider?
Thank you for your patience and for reviewing this information.

Best regards,
Shaoqi

**Comment by biglimp** [comment]
Hopefully @ljarvi or @sunt05 can give you an answer on this.
